pkgs/default.nix

```diff
@@ -3639,24 +3640,31 @@ self: {
           description = "Convert strings into hexadecimal and back";
           license = stdenv.lib.licenses.bsd3;
         }) {};
-      hinotify = callPackage ({ async, base, containers, directory, mkDerivation, stdenv, unix }:
+      hfsevents = callPackage ({ Cocoa, CoreServices, base, bytestring, cereal, mkDerivation, mtl, stdenv, text }:
       mkDerivation {
-          pname = "hinotify";
-          version = "0.3.9";
-          sha256 = "16fzql0s34my9k1ib4rdjf9fhhijkmmbrvi148f865m51160wj7j";
+          pname = "hfsevents";
```

Comment:
@adinapoli-iohk ive regenerated the file on linux
the problem appears to be that stack2nix wasnt fully fixed to work right on darwin